src/Node/CaptureNode.php

```diff
@@ -24,7 +24,7 @@ class CaptureNode extends Node
 {
     public function __construct(Node $body, int $lineno, ?string $tag = null)
     {
-        parent::__construct(['body' => $body], ['raw' => false, 'with_blocks' => false], $lineno, $tag);
```

Comment:
`$blocks` should always be passed to the anonymous function as used by many Twig features.